### PR TITLE
Support for next release of Alfalfa (v0.3.0)

### PR DESCRIPTION
**Pull request by nllong**
* renames several fields
* updates dependencies

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,7 +1,16 @@
 Change Log
 ==========
 
-v0.2.0 (Unreleased)
+v0.3.0
+------
+* Upgrade dependencies
+* Release for Alfalfa 0.2.0
```
Comment (TShapinsky) [inline CHANGELOG.rst:7]: This should be alfalfa v0.3.0, right?

Comment (nllong) [reply CHANGELOG.rst:7]: you are right. Updating.

Comment (nllong) [inline CHANGELOG.rst:7]:
```suggestion
* Release for Alfalfa 0.3.0
```

**Comment by TShapinsky** [comment]
I think this looks good to go. I also want to draw your attention to the [minimal_job_refactor](https://github.com/NREL/alfalfa-client/tree/minimal_job_refactor) branch. It takes advantage of the fact that I'm replacing some values for sites with run information in graphql to make fewer changes to the client. It's functionally identical, though.

**Comment by nllong** [comment]
> I think this looks good to go. I also want to draw your attention to the [minimal_job_refactor](https://github.com/NREL/alfalfa-client/tree/minimal_job_refactor) branch. It takes advantage of the fact that I'm replacing some values for sites with run information in graphql to make fewer changes to the client. It's functionally identical, though.

Seems like we can merge these together... unless the casing of the `wait(url, uid, "READY")` matter?

https://github.com/NREL/alfalfa-client/compare/job_refactor...minimal_job_refactor

**Comment by TShapinsky** [comment]
> > I think this looks good to go. I also want to draw your attention to the [minimal_job_refactor](https://github.com/NREL/alfalfa-client/tree/minimal_job_refactor) branch. It takes advantage of the fact that I'm replacing some values for sites with run information in graphql to make fewer changes to the client. It's functionally identical, though.
> 
> Seems like we can merge these together... unless the casing of the `wait(url, uid, "READY")` matter?
> 
> [job_refactor...minimal_job_refactor](https://github.com/NREL/alfalfa-client/compare/job_refactor...minimal_job_refactor)

The diff it's showing is a bit weird, as they both have the status string changes and the `get_error_log` function and `AlfalfaException`. I'd just leave it at this since we'll be rewriting a bunch of this with the new API anyways.

**Pull request merged**